docs/unittesting.md

```diff
@@ -1,40 +1,35 @@
-# Unit Testing Plugin
+# Solidity Unit Testing Plugin
 
-Click the
-![](images/a-user-testing-icon.png) (double check)
-icon from icon bar to move to the `Solidity Unit Testing` plugin.
+The Solidity Unit Testing plugin allows you to write and run automated tests for your Solidity smart contracts to verify they work correctly before deployment.
 
-If you haven't used this plugin before and are not seeing the `double check` icon, you have to activate it from Remix plugin manager.
+It helps you generate test files, write test functions using a built-in assertion library (like `Assert.equal`, `Assert.notEqual`), and execute those tests in a separate environment to see pass/fail results.
 
-Go to the plugin manager by clicking the ![](images/a-plug.png) (plug) icon and activate `Solidity Unit Testing` plugin.
+## Enabling the Solidity Unit Testing Plugin
 
-![](images/a-unit-testing-from-pm.png)
+To enable the plugin, go to the Plugin Manager, search "Unit Testing" and enable the "Solidity Unit Testing" plugin.
 
-Now the `double check` icon will appear on the left side icon bar. Clicking on the icon will load the plugin in the Side Panel.
+![Enable Solidity Unit Testing plugin](images/unit-testing/activate-test-plugin.gif)
```

Comment:
This .gif is helpful if you don't know how to activate a plugin.  But it seems like that kind of instruction doesn't really belong here - it might be more helpful in the navigating Remix page (/layout/).  I mean it doesn't hurt here- except its a bit distracting.